.github/workflows/build.yml

```diff
@@ -21,28 +21,22 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3, 8.0]
+                php: [8.0, 7.4]
                 symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.9.0, ~1.10.0]
-                node: [10.x]
-                mysql: [5.7]
+                sylius: [~1.9.0, ~1.10.0, ~1.11.0]
```

Comment:
In `composer.json` you also support `1.8`, but there's no version in action.

Reply:
Changed